Improve docs of Swiss public transport for new options (#35609)
home-assistant/home-assistant.github.io: source/_integrations/swiss_public_transport.markdown

The public timetables are coming from [Swiss public transport](https://transport.opendata.ch/).

## Set up a connection

> The minimum configuration for a connection requires a _start_ and _end_ station (e.g., "Zürich HB").

Reviewer: ```suggestion
The minimum configuration for a connection requires a _start_ and _end_ station (for example, "Zürich HB").
```

Optionally, you can provide up to 5 additional _via_ stations.

![Config flow](/images/integrations/swiss_public_transport/config_flow.png)

### Time mode

> The _Time mode_ allows for control over the time of the connections being displayed.

Reviewer: ```suggestion
The _Time mode_ allows you to specify the time of the connections.
```

> - Now (default): Provide the next connections

Reviewer: ```suggestion
- Now (default): Provide the next connections.
```

> - At a fixed time of day: Provide the connections which depart at a fixed time of day (e.g. 18:15 in the evening)

Reviewer: ```suggestion
- At a fixed time of day: Provide the connections that depart at a fixed time of day (for example, 18:15 in the evening)
```

> - At an offset from now: Provide the next possible connections which depart after a specified offset (e.g., 15 minutes from now, which helps account for a 15-minute walk to the station)

Reviewer: ```suggestion
- At an offset from now: Provide the next possible connections which depart after a specified offset (for example, 15 minutes from now, which helps account for a 15-minute walk to the station)
```
we don't usually use [e.g.](https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/e/eg)

![Time mode option](/images/integrations/swiss_public_transport/config_flow_time_mode.png)

#### Use case: Next connection at my local bus stop

Usually, it takes some time to walk to the closest bus station from home, which can be modeled using the `offset` option in the _Time mode_ dropdown, filtering out connections which are not reachable anymore. This information can then be displayed at the door or on your mobile device.

![Offset time mode option](/images/integrations/swiss_public_transport/config_flow_time_offset.png)

#### Use case: Daily train home

> For people working business hours, a set-up using the `fixed` option in the _Time mode_ allows you to identify the same train each day for commuting home. Set up automation to send a push notification to get informed early.

Reviewer: ```suggestion
For people working business hours, a set-up using the `fixed` option in the _Time mode_ allows you to identify the same train each day for commuting home. Set up an automation to send a push notification to get informed early.
```

![Fixed time mode option](/images/integrations/swiss_public_transport/config_flow_time_fixed.png)

> ### Departure vs Arrival

Reviewer: ```suggestion
### Departure versus Arrival
```

Usually, one wants to know when a connection **departs** (default), but in case where the opposite is needed and one wants to know when a connection **arrives**, select "Show arrival time at end station" in the time reference dropdown.

When configured for arrival time, the sensor will display the arrival time at the destination instead of the departure time from the start station. This is particularly useful for automations that need to trigger based on arrival times.

> ![Departure vs arrival option](/images/integrations/swiss_public_transport/config_flow_departure_arrival.png)

Reviewer: ```suggestion
![Departure versus arrival option](/images/integrations/swiss_public_transport/config_flow_departure_arrival.png)
```
